src/briefcase/windows_icon.py

```diff
@@ -0,0 +1,92 @@
+from ctypes import *
+from win32ctypes.pywin32 import win32api
+
+
+# Structures in ico file
+class IconHeader(Structure):
+    _fields_ = [
+        ("Width", c_byte),
+        ("Height", c_byte),
+        ("Colors", c_byte),
+        ("Reserved", c_byte),
+        ("Planes", c_ushort),
```

Comment:
At column 22: (F405) 'c_byte' may be undefined, or defined from star imports: ctypes